src/Map/src/Bridge/Google/src/Option/FullscreenControlOptions.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google\Option;
+
+/**
+ * Options for the rendering of the fullscreen control.
+ *
+ * @see https://developers.google.com/maps/documentation/javascript/reference/control#FullscreenControlOptions
+ *
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final class FullscreenControlOptions
+{
+    public function __construct(
+        public ControlPosition $position = ControlPosition::INLINE_END_BLOCK_START,
```

Comment:
Does this need to be "public" ?

Reply:
Not at all, and we can even makes them readonly